### ISSUE TITLE
Server Statistics aren't loading.

### ISSUE THREAD

**Issue by nacjm** [top-gg/issues#783]
#### Expected Behavior

Expected server vote statistics to show up

#### Observed Behavior
Votes aren't incrementing in the statistics page.
#### Applicable Links or Attachments

![picture of my servers page showing 3 votes](https://user-images.githubusercontent.com/7560033/129196152-74755569-376e-4d54-81a3-d55ef34ed616.png)

#### Steps to Reproduce

1. Open top.gg/server
2. open your servers analytics page after people have voted
3. see that it still says no votes

#### Web Browser and Version

Microsoft Edge Version 92.0.902.67

#### Operating System and Version

Windows 10 19043

#### Screen Resolution

1920x1080

#### Additional Information/Comments

It might be because they just voted a few hours ago.

**Comment by nacjm** [comment]
related to this bug: my servers shows I have 9 votes: 
![image](https://user-images.githubusercontent.com/7560033/129318826-c0f7a64f-9e2c-49e9-ac60-d655ec45fcbd.png)
but when going to the actual server page it says I only have 3. it also isn't having my reply to one of my reviews showing: 
![image](https://user-images.githubusercontent.com/7560033/129318962-cbaea0be-415e-48bb-afd7-86f49fd42bd2.png)

**Comment by velddev** [comment]
> related to this bug: my servers shows I have 9 votes:
> ![image](https://user-images.githubusercontent.com/7560033/129318826-c0f7a64f-9e2c-49e9-ac60-d655ec45fcbd.png)
> but when going to the actual server page it says I only have 3. it also isn't having my reply to one of my reviews showing:
> ![image](https://user-images.githubusercontent.com/7560033/129318962-cbaea0be-415e-48bb-afd7-86f49fd42bd2.png)

Not related, this is the way we cache our responses from the API per web server

**Comment by Xignotic84** [comment]
Are you still having issues with this?

**Comment by nacjm** [comment]
Yes Actually. Views is working now, but the vote tracker is not: 
![image](https://user-images.githubusercontent.com/7560033/133925758-b34d113d-9ad4-4262-b54b-10d1847e7a02.png)
I know I have votes because I have a bot tracking that part for me since the analytics dashboard won't show them
